docs/ref/applications.txt

```diff
@@ -239,12 +239,10 @@ Methods
     as registering signals. It is called as soon as the registry is fully
     populated.
 
-    You cannot import models in modules that define application configuration
-    classes, but you can use :meth:`get_model` to access a model class by
-    name, like this::
-
-        def ready(self):
-            MyModel = self.get_model('MyModel')
+    This is the first point in the application loading process where it's safe 
+    to import models (see :ref:`how-applications-are-loaded` for more details).
+    You can do so with a regular ``import`` statement or by using
```

Comment:
This wording doesn't make it clear that you cannot import models at module-level. In fact, you lost the first sentence, which was the most important information in that paragraph.

> You cannot import models in modules that define application configuration classes.